src/_common/sticker/target/target-controller.ts

```diff
@@ -26,6 +26,19 @@ export class StickerTargetController {
 
 	private _shouldShow = false;
 
+	get canPlaceStickers(): boolean {
```

Comment:
Let's use the disabled prop for this.

We should make functions in StickerPlacement that can check `canPlaceOnPost` and `canPlaceOnComment`. The comment one should be able to take a parentComment too and should literally just call into `canCommentOnModel` for now. The post one should be this code.

You should use the same functions for checking if we should show the sticker button control as well for both comments and posts.